tests/usdt_sample/usdt_sample.md

```diff
@@ -0,0 +1,28 @@
+Tested on Fedora25 4.11.3-200.fc25.x86_64, gcc (GCC) 6.3.1 20161221 (Red Hat 6.3.1-1)
+
+Next to the bcc requirements, the following package(s) are required to build this sample:
+$ sudo dnf install systemtap-sdt-devel  # For Fedora25, other distro's might have differently named packages.
```

Comment:
This will be require an additional package. Also here, the application prints out the latency of the operation. There is no checking whether the test is passed or not. In my opinion, this probably should belong to "example" instead of "test".

We have a unit test test_usdt.py, which you may take a look. It uses folly usdt header file and avoid to use the systemtap-sdt-devel package. I suggest you take a look. This way, we can avoid all these "dtrace" stuff.